### DISCUSSION TITLE
How to set airflow xcom sidecar container image pull policy to IfNotPresent

### DISCUSSION THREAD

**Discussion by okulbida** [apache/airflow#37389] [Q&A] [answered]
Is it possible to set airflow xcom sidecar container image pull policy to `IfNotPresent`. Don't see anything relevant here https://github.com/apache/airflow/blob/main/airflow/providers/cncf/kubernetes/hooks/kubernetes.py

**Comment by y0zg** [comment, marked as answer]
in helm chart
```
airflowLocalSettings: |-
  from airflow.providers.cncf.kubernetes.utils.xcom_sidecar import PodDefaults
  PodDefaults.SIDECAR_CONTAINER.image_pull_policy = "IfNotPresent"
```
